ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -34,7 +34,7 @@
 
 - name: Register mount configuration
   sudo: yes
-  shell: "tail /etc/mtab -n1"
+  shell: "cat /etc/mtab | grep '/opt/data' | sed -e 's/ rw/ user,noauto,rw/'"
```

Comment:
could you use [lineinfile](http://docs.ansible.com/ansible/lineinfile_module.html)? it looks like you're trying to change which users have access to `/opt/data` in that file, which seems like you could use that module